media/mediaType.go

```diff
@@ -167,7 +167,7 @@ var (
 	JSXType        = newMediaType("text", "jsx", []string{"jsx"})
 
 	JSONType           = newMediaType("application", "json", []string{"json"})
-	WebAppManifest = newMediaTypeWithMimeSuffix("application", "manifest", "json", []string{"webmanifest"})
+	WebAppManifestFormat = newMediaTypeWithMimeSuffix("application", "manifest", "json", []string{"webmanifest"})
```

Comment:
This needs to be `WebAppManifestType` (see the other types).

Reply:
Done